tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -334,6 +341,10 @@ def send_datagram(self, data: bytes) -> None:
 
         :param data: The data to send.
         """
+        if not self._protocol._allow_datagrams:
+            _logger.Warn(
```

Comment:
warn()

Also, maybe logging session_id?

Reply:
Done. I don't think logging session_id is useful because the handler author doesn't provide the ID.